internal/operators/mce/config.go

```diff
@@ -13,6 +32,48 @@ const (
 	SNOMinimumCpu    int64 = 8
 )
 
-type Config struct {
-	MceMinOpenshiftVersion string `envconfig:"MCE_MIN_OPENSHIFT_VERSION" default:"4.10.0"`
+func getMinMceOpenshiftVersion(ocpMceVersionMap []OcpMceVersionMap) (*string, error) {
+	lowestVersion := ocpMceVersionMap[0].OpenshiftVersion
+	for _, version := range ocpMceVersionMap {
+		isLower, err := common.BaseVersionLessThan(lowestVersion, version.OpenshiftVersion)
+		if err != nil {
+			return nil, err
+		}
+		if isLower {
+			lowestVersion = version.OpenshiftVersion
+		}
+	}
+
+	return &lowestVersion, nil
+}
+
+func parseMCEConfig(config EnvironmentalConfig) (*Config, error) {
+	var parsedMatrix []OcpMceVersionMap
+	if err := json.Unmarshal([]byte(config.OCPMCEVersionMap), &parsedMatrix); err != nil {
+		return nil, fmt.Errorf("failed to unmarshal ocp mce release matrix: %w", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("failed to unmarshal ocp mce release config: %w", err)
```
remove all reference to "matrix" to avoid any confusions since we replaced it with "map" or "config"